Nos/NosApp.swift

```diff
@@ -21,6 +21,7 @@ struct NosApp: App {
         // hack to fix confirmationDialog color issue
         // https://github.com/planetary-social/nos/issues/1064
         UIView.appearance(whenContainedInInstancesOf: [UIAlertController.self]).tintColor = .systemBlue
+        persistenceController.scheduleBackgroundCleanupTask()
```

Comment:
For some reason, I'm getting a crash on this line. I took a screen shot further down the stack trace in PersistenceController:

<img width="830" alt="Screenshot 2024-10-03 at 4 16 25 PM" src="https://github.com/user-attachments/assets/cc40af80-03b5-4880-afd1-01d8bc2ccb8d">